include/odp/api/spec/crypto.h

```diff
@@ -363,6 +363,10 @@ int odp_crypto_op(const odp_packet_t pkt_in[],
  * creation on the packets. Behaves otherwise like odp_crypto_op() but
  * returns output packets through events.
  *
+ * With other operation types than ODP_CRYPTO_OP_TYPE_LEGACY, packet
```

Comment:
How about

"With operation types other than..."

Reply:
Yes, that is better english. Will change in the next version.